nevergrad/functions/control/test_mujoco.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+import typing as tp
+from .mujoco import GenericMujocoEnv
+from ..base import ExperimentFunction
+from nevergrad.parametrization import parameter as p
+
+
+def test_gym() -> None:
+    class MountainCarContinuous(ExperimentFunction):
```

Comment:
please move it outside the function, I tend to avoid inline class definitions whenever possible (inline definitions tend to go messy when pickling is involved, even though that should not be the case here, I don't see any reason to have it inline anyway)